src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/FastsettePeriodeGrunnlagImpl.java

```diff
@@ -81,6 +81,11 @@ public boolean isSamtykke() {
         return regelGrunnlag.getRettOgOmsorg().getSamtykke();
     }
 
+    @Override
+    public Boolean harOmsorg() {
```

Comment:
Like greit med primitive boolean? Er vel enten eller

Reply:
Boolean for expand/contract. Men kan forsåvidt fjerne alt allerede nå.